smif/schema/pre_specified_schema.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@tomalrussell - You might have to help me out with the schema.  A legal file should look like this:
```yaml
- asset: water_asset_a
  description: Existing water treatment plants
  location: 
    lat: 51.74556
    lon: -1.240528
  new_capacity:
    unit: Ml/yr
    value: 6
  timeperiod: 2010
```